templates/libreswan.connection.secrets

```diff
@@ -1,6 +1,14 @@
 #jinja2: lstrip_blocks: True
 {% for connection in vpn_connections %}
 {%   if item in connection.hosts %}
+{%     if connection.enc_type == 'psk' %}
 @{{ ansible_host | d(inventory_hostname) }} @{{ hostvars[item].ansible_host | d(item) }} : PSK "{{ connection.shared_key_content }}"
+{%     elif connection.enc_type == 'cert' %}
+{%       for hostname in connection.hosts %}
+{%         if hostname == inventory_hostname or hostname == ansible_host %}
+@{{ ansible_host | d(inventory_hostname) }} @{{ hostvars[item].ansible_host | d(item) }} : RSA "{{ connection.hosts[hostname].cert_name | d('') }}"
```

Comment:
If `cert_name` is not specified, what happens?  If the value in the file is `RSA ""`, does ipsec report an error?  If so, what is the error?  If `enc_type` is `cert` and `cert_name` is not specified, the error should indicate that the cert is empty or missing.

Reply:
![image](https://user-images.githubusercontent.com/17239268/100757417-5b167a00-33bc-11eb-9890-9527a73510ef.png)
This is the error that ipsec reports when `cert_name` is not specified and the secrets file lists `RSA ""`. Is saying that the certificate is invalid specific enough, or should it more specifically indicate that the cert name is empty? We could always add a check in Ansible to validate that the `cert_name` fields are populated when `enc_type` is `cert`.

Reply:
> We could always add a check in Ansible to validate that the cert_name fields are populated when enc_type is cert

If it is simple to do, then ok.

Reply:
This extra check is included in the latest commit.